.github/workflows/android-build.yml

```diff
@@ -0,0 +1,35 @@
+name: Android build
+
+on:
+  pull_request:
+
+# Cancel previous runs if a more recent commit is pushed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  android-build:
+    name: Check Android build with gradle
+    runs-on: "ubuntu-20.04"
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v3
+        with:
+          submodules: 'true'
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Setup Vulkan SDK
+        run: |
+          wget -q https://sdk.lunarg.com/sdk/download/1.3.216.0/linux/vulkansdk-linux-x86_64-1.3.216.0.tar.gz
+          mkdir "${HOME}/vulkan-sdk"
+          tar -xf vulkansdk-linux-x86_64-1.3.216.0.tar.gz -C "${HOME}/vulkan-sdk"
+          echo "VULKAN_SDK=${HOME}/vulkan-sdk/1.3.216.0/x86_64" >> $GITHUB_ENV
+      - name: Build Cube sample for Android
+        run: |
+          ./gradlew assembleMobileGbufferDebug
```

Comment:
Why only build 1 sample? is there a way to query what rules are available to build all the samples?

Reply:
I updated it to build all samples for mobile only. This job seems to take about ~15 minutes which is slightly longer than what windows and linux builds take. Including Openxr targets increases the total time to ~30 minutes. Quest targets rely on the oculus OpenXR loader, which we'll have to figure something out for if we wanted to test those as well. 
We could do mobile and openXr now, but I kinda don't like the idea of increasing the CI time. Thoughts?